geoips/plugins/modules/readers/amsr2_netcdf.py

```diff
@@ -285,7 +285,8 @@ def read_amsr_data(full_xarray, chans):
     return xarrays
 
 
-def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=False):
+def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=False,
```

Comment:
I may have accidentally added this... Should we leave it in order to test for now...?

Reply:
Can we collect these in an issue that talks about adding more appropriate tests so that we don't forget to remove these later?

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/334
I did